pkgs/by-name/ea/earbuds/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  fetchpatch,
+  installShellFiles,
+  nix-update-script,
+  pkg-config,
+  dbus,
+  libpulseaudio,
+  bluez,
+}:
+rustPlatform.buildRustPackage {
+  pname = "earbuds";
+  version = "0.1.9-unstable-2024-06-28";
+
+  src = fetchFromGitHub {
+    owner = "JojiiOfficial";
+    repo = "LiveBudsCli";
+    rev = "df46706e44fa9e7de355b11eab4cc850efe968a3";
+    hash = "sha256-IEor7aZnwCA6Rg2gXIYSQ65hV/jJOKehujOSZnVzVis=";
+  };
+
+  patches = [
+    # fix daemon autostart not working
+    (fetchpatch {
```

Comment:
Better to include patch here since upstream author may require you to force push.

Reply:
Also, for the reference, I think it is better to use `fetchpatch2` from now on.
Anyway, I agree with @Aleksanaa about embedding the patch in nixpkgs.